src/ROOTex.jl

```diff
@@ -45,6 +45,22 @@ function Fit(g::TGraph, tf1::TF1, option::String = "", goption::String = "", rxm
     Fit(g, CxxPtr(tf1), option, goption, rxmin, rxmax)
 end
 
+#---TFile extensions-------------------------------------------------------------------------------
+function GetObject(file::Union{ROOT.TDirectoryFile, CxxPtr{<:ROOT.TDirectoryFile}}, name::AbstractString)
+    obj = Get(file, name)               # Get object from file (TObject*)     
+    obj == C_NULL && return C_NULL
+    type = getproperty(ROOT, IsA(obj) |> GetName |> Symbol)
```

Comment:
This will work only if `obj`  derives from `TObject`. My guess is that it will even crash if it doesn't.

I expect we should use:
* `DecodeNameCycle` to extract name and cycle
* `k = GetKey(dir, name, cycle)`
* `GetClassName(k)`

Note for this we should add 'TKey' and 'TDirectory' to export to Julia above functions.

We can keep using `GetObject` (instead of TKey::Readobj`) as it looks first for the object into memory.

Reply:
Proposal implemented.